services/mock-service/src/config.ts

```diff
@@ -38,8 +40,4 @@ export const API_PATHS = {
     'financial-service-providers/safaricom/timeout-callback',
 };
 
-export const EXTERNAL_API_ROOT = `${
-  DEVELOPMENT
-    ? `http://121-service:${process.env.PORT_121_SERVICE}/`
-    : process.env.EXTERNAL_121_SERVICE_URL
-}api`;
+export const EXTERNAL_API_ROOT = `${env.EXTERNAL_121_SERVICE_URL}api`;
```

Comment:
Needs a heads-up to devs that they possibly need to update their .env locally?

Reply:
Theoretically, the heads-up will be given by the `t3-env`-tool on startup of the Mock-Service...

That should throw the error about a possibly faulty value of `EXTERNAL_121_SERVICE_URL`, and thus pointing to the 'requirements' defined in `src/env.ts`.

So, the idea is, that no "reading the documentation first" would be required.

---

Also, technically the values `http://localhost:3000` and `http://121-service:3000` should work the same. (But in practice, as seen on this PRs' test-runs, not for all environments/tools. 😢 )
In this specific refactoring, I tried to minimise the number of:
- ENV-variables that need to be set/handled. 
- "variants of some URL of the 121-service".
- "arbitrary switches based on `NODE_ENV`".